### ISSUE TITLE
On mobile, the voting buttons float above the term and definition

### ISSUE THREAD

**Issue by machawk1** [metadata-research/yamz#194]
**Describe the bug**
In the production version of YAMZ at yamz.net, visiting a term produces on mobile produces a display where the voting buttons float above other design elements.
![Screenshot_20241008-085456](https://github.com/user-attachments/assets/27ede465-4fef-40a2-a04b-93525af3d1d9)

**To Reproduce**
*Steps to reproduce the behavior:*  
1. Visit a term at yamz.net on mobile.
1. Note the voting buttons floating above other elements.

**Expected Behavior**  
I am not sure what the expected behavior is here on mobile. On desktop, the voting buttons are to the left of the term and the definition, but this might not be visually intuitive on mobile.

**Smartphone:**  
Android, Brave Browser

**Comment by jkunze** [comment]
Good point. I just assigned two issues to @ns3363, and we should do this routinely, given that Naima is working on them.
